tests/hub/slice_controller_test.go

```diff
@@ -187,6 +198,9 @@ var _ = Describe("Hub SliceController", func() {
 				return errors.IsNotFound(err)
 			}, time.Second*10, time.Millisecond*250).Should(BeTrue())
 
+			m := utils.GetCounterMetricFromRegistry(MetricRegistry, "kubeslice_slice_deleted_total")
+			Expect(m).To(ContainElement(1.0))
```

Comment:
Don't we have ITs for Slice Delete to captured that event too?

Reply:
Only metrics and related test cases are covered as part of this PR

Reply:
Can you please provide the estimates for adding these testcases

Reply:
I'm not sure. I'm only looking into metrics currently, not events.